app/styles/ui/_welcome.scss

```diff
@@ -62,15 +66,21 @@
   }
 
   input,
-  button {
+  button,
+  select {
     font-size: var(--font-size-md);
-    height: auto;
   }
 
   input {
     padding: var(--spacing-half);
   }
 
+  select {
+    // We need to tweak the horizontal padding a bit here to fit correctly with
+    // the bigger size
+    padding: 0 1px;
+  }
+
```

Comment:
The changes in this file are what fix the visual glitch from #11561 

The problem was that the bigger size of text fields and buttons relied entirely on a bigger font size (`--font-size-md` instead of the default `--font-size`) and `auto` height. For _reasons_, just increasing the font size doesn't impact the `select`'s height the same way, so it needs some help 😆 

Since we already have some nice height variables that were used by buttons, inputs and selects, I basically overrode them in the `welcome` element with the value we need, `29px`.

Additionally, the `select` element requires a small tweak in the horizontal padding to make it match the one from the text box.

To be honest, I can't tell which approach is hackier: what we have now or what we had before 😂  Maybe they're both about the same level of hackiness?? 🧙

Reply:
Hackiness is the world of html components and css sometimes... 😂